docs/source-pytorch/cli/lightning_cli_advanced.rst

```diff
@@ -126,6 +126,53 @@ which generates a config like:
         # Fit your model using the edited configuration
         python main.py fit --config config.yaml
 
+Configuration items can be either simple Python objects such as int and str,
+or complex objects comprised of a ``class_path`` and ``init_args`` arguments. The ``class_path`` refers
+to the complete import path of the item class, while ``init_args`` are the arguments to be passed
```

Comment:
The complete import path is not required because the CLI auto-registers all imported modules so that they can be used with just the Module name

Reply:
But it is required for other classes not inherited from `pytorch_lightning.Module` class. I can add a note about it